.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# All website content
+content/ @jenkins-infra/copy-editors
```

Comment:
any reason this team is secret?

if it's secret they won't show up in the pull request showing that review has been requested and it will just say waiting on codeowner review without saying who

Reply:
I do not know, the team was created eons ago. Maybe @olblak or @rtyler remember reasons why it is secret

Reply:
I don't see why it's private and there is no reason to keep this way. 
That team only has access to two public repositories

Reply:
I am changing to public

Reply:
Heeu that team is already visible

Reply:
assume you need to be a member of the org to see it then =/, I can't mention them.